scene/resources/particle_process_material.cpp

```diff
@@ -419,7 +419,7 @@ void ParticleProcessMaterial::_update_shader() {
 	if (tex_parameters[PARAM_ANGLE].is_valid()) {
 		code += "	float tex_angle = textureLod(angle_texture, vec2(0.0, 0.0), 0.0).r;\n";
 	} else {
-		code += "	float tex_angle = 0.0;\n";
+		code += "	float tex_angle = 1.0;\n";
```

Comment:
I notice that CPUParticles also used in turns `0.0` and `1.0` like ParticlesProcessMaterial, should they be changed too?
```
$ rg 'tex_angle ='
scene/3d/cpu_particles_3d.cpp
745:                    real_t tex_angle = 0.0;
747:                            tex_angle = curve_parameters[PARAM_ANGLE]->sample(tv);
945:                    real_t tex_angle = 1.0;
947:                            tex_angle = curve_parameters[PARAM_ANGLE]->sample(tv);

scene/2d/cpu_particles_2d.cpp
725:                    real_t tex_angle = 0.0;
727:                            tex_angle = curve_parameters[PARAM_ANGLE]->sample(tv);
861:                    real_t tex_angle = 1.0;
863:                            tex_angle = curve_parameters[PARAM_ANGLE]->sample(tv);

scene/resources/particle_process_material.cpp
420:            code += "       float tex_angle = textureLod(angle_texture, vec2(0.0, 0.0), 0.0).r;\n";
422:            code += "       float tex_angle = 0.0;\n";
638:            code += "       float tex_angle = textureLod(angle_texture, vec2(tv, 0.0), 0.0).r;\n";
640:            code += "       float tex_angle = 1.0;\n";
```